### DISCUSSION TITLE
Feat: Estimated time required to read and finish blog.

### DISCUSSION THREAD

**Discussion by shashankbhosagi** [SahilK-027/sahilk027.blog#2] [Ideas]
At the top of the blog please add a feature that mentions the `estimated time` it would take an average person to complete reading the blog. Eg. `12mins` `20mins`.
@SahilK-027

**Comment by SahilK-027** [comment]
Sure, I will add it 👍🏻.
Thanks for the recommendation 🙇🏻 @shashankbhosagi !

**Comment by SahilK-027** [comment]
This is added on Prod.
cc: @shashankbhosagi
